debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -1220,6 +1220,11 @@ STRING_USER_NAME:                    (
                                      (
                                        SQUOTA_STRING | DQUOTA_STRING 
                                        | BQUOTA_STRING | ID_LITERAL
+                                       | IP_ADDRESS
+                                     );
+IP_ADDRESS:                          (
+                                       [0-9]+ '.' [0-9.]+
```

Comment:
Isn't it always four numbers? I know it doesn't matter too much, but I think I'd give it simply as that: `[0-9]+ '.' + [0-9]+ '.' [0-9] + '.' + [0-9]`.